To: vim_dev@googlegroups.com
Subject: Patch 7.3.206
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.206
Problem:    64bit MS-Windows compiler warning.
Solution:   Use HandleToLong() instead of type cast. (Mike Williams)
Files:	    src/gui_w32.c


*** ../mercurial/vim73/src/gui_w32.c	2011-01-17 20:08:04.000000000 +0100
--- src/gui_w32.c	2011-05-25 21:14:10.000000000 +0200
***************
*** 1575,1581 ****
  
  #ifdef FEAT_EVAL
      /* set the v:windowid variable */
!     set_vim_var_nr(VV_WINDOWID, (long)s_hwnd);
  #endif
  
  theend:
--- 1575,1581 ----
  
  #ifdef FEAT_EVAL
      /* set the v:windowid variable */
!     set_vim_var_nr(VV_WINDOWID, HandleToLong(s_hwnd));
  #endif
  
  theend:
*** ../vim-7.3.205/src/version.c	2011-05-25 17:56:23.000000000 +0200
--- src/version.c	2011-05-25 21:16:17.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     206,
  /**/

-- 
FIXME and XXX are two common keywords used to mark broken or incomplete code
not only since XXX as a sex reference would grab everybody's attention but
simply due to the fact that Vim would highlight these words.
					-- Hendrik Scholz

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
